v2/src/log_context_property_basic_types.c

```diff
@@ -13,149 +14,209 @@
-        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_031: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint32_t).to_string shall fail and return a negative value. ]*/ \
-        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_041: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IMPL(int16_t).to_string shall fail and return a negative value. ]*/ \
-        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_051: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint16_t).to_string shall fail and return a negative value. ]*/ \
-        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_061: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IMPL(int8_t).to_string shall fail and return a negative value. ]*/ \
-        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_071: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint8_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_001: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int64_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_011: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint64_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_021: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int32_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_031: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint32_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_041: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int16_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_051: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint16_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_061: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int8_t).to_string shall fail and return a negative value. ]*/ \
+        /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_071: [ If property_value is NULL, LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint8_t).to_string shall fail and return a negative value. ]*/ \
         if (property_value == NULL) \
         { \
-            (void)printf("Invalid arguments: void* property_value=%p, char* buffer=%p, size_t buffer_size=%zu\r\n", \
-                property_value, buffer, buffer_size); \
+            (void)printf("Invalid arguments: void* property_value=%p, char* buffer=%p, size_t buffer_length=%zu\r\n", \
+                property_value, buffer, buffer_length); \
             result = -1; \
```

Comment:
>1 [](http://example.com/codeflow?start=22&length=1)

Can we have instead of "-1" -> "-MU_FAILURE"? Generally talking it miiiiight produce different numbers should we ever have to err... debug this.

Reply:
I'll leave it as negative as we return the sprintf result, so I will stick to that scheme.